superset/db_engine_specs/hive.py

```diff
@@ -173,12 +158,13 @@ def convert_to_hive_type(col_type: str) -> str:
         import boto3  # pylint: disable=import-error
 
         s3 = boto3.client("s3")
-        location = os.path.join("s3a://", bucket_path, upload_prefix, table_name)
+        location = os.path.join("s3a://", bucket_path, upload_prefix, table.table)
         s3.upload_file(
             filename,
             bucket_path,
-            os.path.join(upload_prefix, table_name, os.path.basename(filename)),
+            os.path.join(upload_prefix, table.table, os.path.basename(filename)),
         )
+        # TODO(bkyryliuk): support other delimiters
```

Comment:
It seems like `full_table_name` is only used once and thus I wonder if we should annex line 137 and make line 168, 

```python
sql = f"""CREATE TABLE {str(table)} (
```